secantTests/SendTests/SendTests.swift

```diff
@@ -352,6 +352,9 @@ class SendTests: XCTestCase {
     }
 
     func testDifferentNumberFormats_LiveNumberFormatter() throws {
+        let zcashNumberFormatter = NumberFormatter.zcashNumberFormatter
+        zcashNumberFormatter.locale = Locale(identifier: "en_US")
```

Comment:
Why was this change necessary?

Reply:
Yes, the simulators have different locale settings and tests were failing. Also CIs use different version of xcode and simulators so I needed to setup the tests with hardcoded locale to US (sometime to CZ so I can check comma instead of point as a floating separator).